ManikShrivastav.md

```diff
@@ -0,0 +1,50 @@
+---
+name: "ManikShrivastav"
+project: "Gesture Control Robot"
+---
+
+# Gesture Control Robot
+
+
+## Summary
+
+Gesture Robot is an electronic car controlled by the movement of circuit. It can be mostly used in real 
+life robot fights and car fights. It is a car that can be controlled according to the user’s movement of the 
+hands. The transmitter part will be attached to the hand of the user and its receiver will act as the 
+motherboard of the car. As the transmitter moves, it will send the signals to the receiver in the car and it 
+will turn and run the car according to the instructions.
+
+## Plan
+
+First of all, the chassis of car is going to be made. It will contain four motors and rubber wheels attached 
+to the frame of the robot made out of PlyWood. After that, the motors will be connected with motor shield driver. 
```

Comment:
Do you already have plywood you plan to use for this project? I don't see it in your budget.

Reply:
Yes, I already own a plywood for making the project.

Reply:
Great.

Reply:
Once you do so, please request me for another review. It's almost good to go!

Reply:
Done adding plywood to the budget.